io.github.nostar.DroidStar.yml

```diff
@@ -0,0 +1,43 @@
+app-id: io.github.nostar.DroidStar
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: DroidStar
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --socket=pulseaudio
+  - --filesystem=xdg-download
+  - --device=all
+  - --share=network
+modules:
+  - name: imbe_vocoder
+    buildsystem: simple
+    build-commands:
+      - make
+      - install -Dm644 libimbe_vocoder.a $FLATPAK_DEST/lib/libimbe_vocoder.a
+      - mkdir -p $FLATPAK_DEST/include
+      - install imbe_vocoder_api.h  $FLATPAK_DEST/include/imbe_vocoder_api.h
+    sources:
+      - type: git
+        url: https://github.com/nostar/imbe_vocoder.git
+        commit: 03423cb00b8e9ba1be164493e8b98b5bd7c1508c
+  - name: DroidStar
+    buildsystem: simple
+    build-commands:
+      - qmake LIBS+=-L$FLATPAK_DEST/lib DroidStar.pro
+      - make -j`nproc`
+    post-install:
+      - install -D io.github.nostar.DroidStar.metainfo.xml $FLATPAK_DEST/share/metainfo/io.github.nostar.DroidStar.metainfo.xml
+      - install -D DroidStar $FLATPAK_DEST/bin/DroidStar
+      - install -D images/droidstar.png $FLATPAK_DEST/share/icons/hicolor/128x128/apps/io.github.nostar.DroidStar.png
```

Comment:
Upstream icon seems to be 423px × 423px. You first have to convert the icon to 128px x 128px (or 256px x 256px).

Reply:
ok